src/MorganStanley.ComposeUI.MainApp/MainWindow.axaml.cs

```diff
@@ -0,0 +1,23 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using Avalonia.Controls;
+
+namespace MorganStanley.ComposeUI.MainApp
+{
+    public partial class MainWindow : Window
```

Comment:
I can see having this project here now to have a main Avalonia entrypoint project, not sure if having a Window in it makes sense.

Reply:
Yes, I think it will be a shell window.

Reply:
For now we'll be working primarily with the prototype projects. When we get to AvaloniaShellApp, we'll figure out anything else what is needed there. I think I should be a window, hosting some other plugins in it and also allowing to drag them out , dock and tab them.